packages/cli/src/commands/new/project.ts

```diff
@@ -82,8 +82,8 @@ const run = async (flags: Partial<ProjectInitializerConfig>, boosterVersion: str
   Script.init(`boost ${Brand.energize('new')} 🚧`, parseConfig(new Prompter(), flags, boosterVersion))
     .step('Creating project root', generateRootDirectory)
     .step('Generating config files', generateConfigFiles)
-    .optionalStep(Boolean(flags.skipInstall), 'Installing dependencies', installDependencies)
-    .optionalStep(Boolean(flags.skipGit), 'Initializing git repository', initializeGit)
+    .optionalStep(Boolean(!flags.installDependencies), 'Installing dependencies', installDependencies)
+    .optionalStep(Boolean(!flags.initializeGit), 'Initializing git repository', initializeGit)
```

Comment:
Now that I see this change I wonder if it would make more sense for someone using the default behavior to install the packages and install git for them...

As a new user, what would you expect?

Reply:
i agree: i think being able to start writing code right away is the better way to go. you would eventually have to do those steps yourself anyway, so might as well do them during install

Reply:
i have made these changes, let me know what you think!